src/App.vue

```diff
@@ -301,11 +304,32 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     // only initialize once
     // don't start during Jest tests as it messes up the test JWT
     if (this.tokenService || this.isJestRunning) return
+    try {
+      console.info('Starting token refresh service...') // eslint-disable-line no-console
+      this.tokenService = new TokenService()
+      await this.tokenService.init()
+      this.tokenService.scheduleRefreshTimer()
+    } catch (error) {
+      // Happens when the refresh token has expired in session storage
+      // reload to get new tokens
+
+      // eslint-disable-next-line no-console
+      console.log('Could not initialize token refresher: ', error)
+      this.clearKeycloakSession()
+      location.reload()
+    }
+  }
 
-    console.info('Starting token refresh service...') // eslint-disable-line no-console
-    this.tokenService = new TokenService()
-    await this.tokenService.init()
-    this.tokenService.scheduleRefreshTimer()
+  /**
+   * Clears Keycloak token information from session storage
+   */
+  private clearKeycloakSession () : void {
+    sessionStorage.removeItem(SessionStorageKeys.KeyCloakToken)
+    sessionStorage.removeItem(SessionStorageKeys.KeyCloakIdToken)
+    sessionStorage.removeItem(SessionStorageKeys.KeyCloakRefreshToken)
+    sessionStorage.removeItem(SessionStorageKeys.UserFullName)
+    sessionStorage.removeItem(SessionStorageKeys.UserKcId)
+    sessionStorage.removeItem(SessionStorageKeys.UserAccountType)
```

Comment:
Does one of these also clear our app's token? It's used by isAuthenticated() in a couple of places.

`sessionStorage.getItem('KEYCLOAK_TOKEN')`

Reply:
Yes, `SessionStorageKeys.KeyCloakToken`. Should I move away from the constants within the sbc-components?

Reply:
OK I get it.

Actually, since "we" aren't the ones setting these tokens, it's better to use "their" keys to reference them. (Too bad they don't have a util method to clear these eh?)

Would you mind changing over our other references to KEYCLOAK_TOKEN to their keys? (except in the unit tests 😄 )

Reply:
Done!